package/src/index.ts

```diff
@@ -48,8 +70,30 @@ export class CandleClient {
     );
   }
 
-  public async getLinkedAccounts(): Promise<LinkedAccount[]> {
-    return this.candle.getLinkedAccounts();
+  public async getLinkedAccounts(): Promise<
+    (
+      | (LinkedAccount & { state: "active" })
+      | ({ state: "inactive" } & Pick<
+          LinkedAccount,
+          "linkedAccountID" | "service"
+        >)
+    )[]
```

Comment:
#req match HTTP/Swift model structure with nested `details` object
#nth make `type` or `interface` for this return type so it's cleaner in the VSC hover popup